### PR TITLE
Added tooling to support cross-platform compilation

### PR DESCRIPTION
**Pull request by migueldeicaza**

### CODE REVIEW

**Comment by akoeplinger** [comment]
Same comments/questions as in https://github.com/mono/mono/pull/2970#discussion-diff-63276081 😄

**Comment by ludovic-henry** [comment]
Well done for PR # 3000 👍

**Comment by migueldeicaza** [comment]
Alexander, I was planning on using a different servers, but you are correct, this is just another https certificate to maintain.

File: man/mkbundle.1 [hunk 1]
```diff
@@ -21,9 +21,34 @@ default only the assemblies specified in the command line will be
...
+runtime to be installed in the ~/.mono/targets/TARGET/mono to be
+available.   You can use the "--local-targets" to list all available
+targets, and the "--cross" argument to specify the target, like this:
+.nf
+	$ mkbundle --local-targets	
+	Available targets:
+		default	- Current System Mono
+		4.4.0-macosx-x86
+		4.4.0-debian-8-arm64
+	$ mkbundle --cross 4.4.0-debian-8-powerpc hello.exe -o hello-debian
```
Comment (akoeplinger) [inline man/mkbundle.1]: `4.4.0-debian-8-powerpc` is not listed in `available targets`.

Btw. what are the plans regarding Mono versions here? Do we push all releases there? If yes, should this use the 4part version like 4.4.0.168 instead?

Comment (migueldeicaza) [inline man/mkbundle.1]: That is entirely under our control, and TBD.

**Pull request merged**